spec/concepts/api/v1/lib/service/email_token_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::V1::Lib::Service::EmailToken do
+  let(:payload) { { params: 42 } }
+  let(:hmac_secret) { described_class::HMAC_SECRET }
```

Comment:
For what? Please use stub_const with reference to some let instead, for instance:
```ruby
let(:hmac_secret) { 'hmac_secret' }
```

Or direct call of these ones.

Reply:
Done